desktop-app/src/lib/ledger.ts

```diff
@@ -216,7 +217,7 @@ let sendingWindow = null;
 
 // Create a copy of the ledger service and register listeners with the browser window
 export function initialize() {
-  console.log("Ledger service inializing")
+  console.log('Ledger service inializing');
```

Comment:
seems to be an old typo, can we fix it? ^^

```suggestion
  console.log('Ledger service initializing');
```